CMakeLists.txt

```diff
@@ -48,4 +50,22 @@ target_link_libraries(input-overlay
         ${input-overlay_PLATFORM_DEPS}
         ${UIOHOOK_LIBRARY})
 
-install_obs_plugin_with_data(input-overlay data)
+# --- Linux-specific build settings and tasks ---
+if(UNIX AND NOT APPLE)
+	include(GNUInstallDirs)
+
+	set_target_properties(input-overlay PROPERTIES PREFIX "")
+
+	file(GLOB locale_files data/locale/*.ini)
+
+        set(CMAKE_INSTALL_DEFAULT_DIRECTORY_PERMISSIONS
+		OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_WRITE GROUP_EXECUTE WORLD_READ WORLD_EXECUTE)
+
+	install(TARGETS input-overlay LIBRARY
+		DESTINATION "${CMAKE_INSTALL_FULL_LIBDIR}/obs-plugins"
+		PERMISSIONS OWNER_READ OWNER_WRITE GROUP_READ WORLD_READ)
+
+	install(FILES ${locale_files}
+		DESTINATION "${CMAKE_INSTALL_FULL_DATAROOTDIR}/obs/obs-plugins/input-overlay/locale")
+endif()
```

Comment:
can you readd the ``install_obs_plugin_with_data(input-overlay data)`` part for windows? Otherwise it just won't build at all on windows. Just add
```
else()
    install_obs_plugin_with_data(input-overlay data)
endif()
```
here

Reply:
Hey there. I can do that, but I don't think that'd be quite correct. I've adapted the windows section from obs-websocket. I initially left it out because I couldn't test it. If it doesn't work, I'm happy to fall back on this. I don't think there are many Windows packaging needs that rely on building from source so it probably won't make a huge difference if we can't do it "right".

Reply:
I don't really have time to test it so I'd rather leave it the way it was.

Reply:
Okie doke, fair enough. Fixed. Do you care if we leave the Windows version in the history in case you do want to try it at some point? Otherwise I can try to squash those two commits.

Reply:
it's fine, thanks for the contribution!

Reply:
Thank you likewise!